### ISSUE TITLE
Markbot question

### ISSUE THREAD

**Issue by MJholland** [MJholland/stylish-beets#1]
@thomasjbradley, The text on my site isn't totally formatted the same as the screen shot but markbot says my code is good to hand in. Just wondering if that is normal? One example is the italics quote, in the screen shot the text is large but in mine it is the same size as the rest of the text. Markbot says it's fine though.

**Comment by thomasjbradley** [comment]
Yeah, Markbot is a little lenient right now on how it looks. It's checking for specific CSS properties to be used but it might not be exactly the same. Soon it will take screenshots of your work and compare against the reference one.
